explainability-service/src/main/java/org/kie/trustyai/service/endpoints/metrics/UniversalListingEndpoint.java

```diff
@@ -0,0 +1,48 @@
+package org.kie.trustyai.service.endpoints.metrics;
+
+import java.util.Map;
+import java.util.UUID;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.jboss.logging.Logger;
+import org.kie.trustyai.service.payloads.BaseMetricRequest;
+import org.kie.trustyai.service.payloads.scheduler.ScheduleList;
+import org.kie.trustyai.service.payloads.scheduler.ScheduleRequest;
+import org.kie.trustyai.service.prometheus.PrometheusScheduler;
+
+@Tag(name = "Metrics Information Endpoint", description = "Return a list of all metrics being currently requested.")
+@Path("/metrics/all")
```

Comment:
To be honest I feel after the TrustyAI operator implementation we should move all of this to K8s objects: every new request (aka metrics registered) can become a `TrustyAIRequest` (or similar) and this `getAll` is just a plain query to k8s API (`kubectl get trustyairequest -n my-namespace`).
Let's go on with this to unlock the integration but we should revisit because in the long run I would like to avoid any direct REST service invocation from dashboard and just query kubeAPI (i.e. in the new on instance per namespace this implies to have a dynamic mechanism to retrieve TrustyAI service URL).

@ruivieira @tteofili 
Wdyt?

Reply:
@danielezonca I agree on both points: going with this PR for now and use the mechanism you propose going forward with the operator.